src/SearchField/__stories__/index.js

```diff
@@ -0,0 +1,104 @@
+/**
+ *  Copyright 2020 Adobe. All rights reserved.
+ *  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License. You may obtain a copy
+ *  of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under
+ *  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *  OF ANY KIND, either express or implied. See the License for the specific language
+ *  governing permissions and limitations under the License.
+ */
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import Magnify from '@spectrum-icons/workflow/Magnify'
+import Close from '@spectrum-icons/workflow/Close'
+
+import '@spectrum-css/icon'
+import '@spectrum-css/search'
+import '@spectrum-css/textfield'
+import '@spectrum-css/vars'
```

Comment:
Do you have duplicate spectrum css vars if you pull it together with the @react-spectrum/Provider in the css build file ?

Reply:
I checked the `package.json` of spectrum-css/vars and main refers to `dist/spectrum-metadata.json` so I think you can remove this since you're using the vars pulled by the Provider.

Reply:
👍